tests/mocks/mock_db.py

```diff
@@ -1,4 +1,30 @@
+import csv
+import pandas as pd
 
 
 class MockDB:
-    pass
+
+    phases = {'N/A': 0, 'Phase 2': 0, 'Phase 3': 0, 'Phase 4': 0}
+    studies = None
+    file = None
+
+    def __init__(self, filename):
+        self.file = filename
+
+    def populate_tables(self):
+        with open(self.file) as file:
```

Comment:
Pandas has a [read_csv](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html) function that will read the data directly into a `DataFrame`.  Using that function will result in a one-line implementation here.

Not sure what you are trying to accomplish in the for loop...

Reply:
BTW: In Pandas we refer to columns by name rather than counting to 38...